dist/css/main.css

```diff
@@ -68,7 +68,7 @@
   .menu .nav-link {
     display: inline-block;
     position: relative;
-    font-size: 30px;
+    font-size: 40px;
```

Comment:
Font-size: 40px for navigation menu could be bigger for mobile screens. Also, it could be better as it is not a text-heavy website.